apps/web-giddh/src/app/contact/account-statement/account-statement.component.html

```diff
@@ -0,0 +1,444 @@
+                                                ? element?.voucherAmount?.amount
+                                                : "-"
+                                        }}
+                                    </span>
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Payments">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_payment }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    {{ element?.voucherAmount.type != "DEBIT" ? element?.voucherAmount?.amount : "-" }}
+                                </td>
+                            </ng-container>
+                            <ng-container matColumnDef="Balance">
+                                <th
+                                    mat-header-cell
+                                    *matHeaderCellDef
+                                    class="bg-black text-white sort-header-right text-right font-16 fw-900"
+                                >
+                                    {{ commonLocaleData?.app_balance }}
+                                </th>
+                                <td
+                                    mat-cell
+                                    *matCellDef="let element; let i = index"
+                                    [ngClass]="{ 'bg-light-grey': i % 2 }"
+                                    class="sort-header-right text-right"
+                                >
+                                    {{
+                                        element.closingBalance?.amount >= 0
+                                            ? (element.closingBalance?.type === "CREDIT" ? "-" : "") +
+                                              element.closingBalance.amount
+                                            : ""
+                                    }}
+                                </td>
+                            </ng-container>
+                            <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+                            <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+                        </table>
+                        <hr class="border-grey" />
+                        <table class="d-flex justify-content-end">
```

Comment:
```suggestion
                        <table class="d-flex justify-content-end">
```

Remove this "d-flex justify-content-end" from table tag or use div tag because table have display : table